src/documentation.ts

```diff
@@ -85,8 +85,10 @@ export default class Documentation {
         }
 
         // Get the props signatures
-        details.properties = Object.keys(symbol.members).sort()
-            .map((name) => this.serializeDeclaration(symbol.members[name]));
+        // symbols without a `valueDeclaration` will crash things on TS 2.0, so filter these out
+        details.properties = Object.keys(symbol.members).sort().map((name) => symbol.members[name])
+            .filter((symbol) => symbol.valueDeclaration != null)
+            .map((symbol) => this.serializeDeclaration(symbol));
```

Comment:
does this end up omitting any fields?

these APIs are notoriously undocumented, and I've seen this exact situation with other types (I think this is why classes are currently unsupported), so i'm down for a "whatever works" solution